### DISCUSSION TITLE
Cannot deserialize ByteVector

### DISCUSSION THREAD

**Discussion by xbukorg** [ACINQ/eclair#2544] [Q&A] [answered]
Hi,

How can I fix this frequently observed error?

2022-12-29 12:24:05,615 ERROR f.a.e.c.TransportHandler CON n:03c43ed526a235cf15234e3e8eb34a1519ae4b5b464667fe77ff7af615d41f23aa - cannot deserialize ByteVector(130 bytes, 0x0102fcb2a8327a5b9824153834356e2ea66ced9e3141fc01f70c724627166d9cf82621a5a301868d155a86247ac5750e3001aef7fcbc4f78c9a489f9ccf9f3c9e9636fe28c0ab6f1b372c1a6a246ae63f74f931e8365e15a089c68d619000000000008292a0004ad000063ad7600000000210000000000017d080000109900000002): messageFlags: expected constant BitVector(1 bits, 0x8) but got BitVector(1 bits, 0x0)

Thanx

**Comment by DerEwige** [comment, marked as answer]
I asked the same question in another thread.
See the link the answer:

https://github.com/ACINQ/eclair/discussions/2517#discussioncomment-4290819

TLDR: Channel updates without max_htlc, which should no longer be broadcasted, but still exist, because of old nodes

**Reply by xbukorg** [reply]
Hi,

Thanks for a info. 

When I understand it can be fixed in eclair.config by this way:
sync-whitelist = [] // a list of public keys; if non-empty, we will only do the initial sync with those peers

and Eclair have defined missing parameters by this way:

// The following parameters apply to each HTLC direction (incoming or outgoing), which means that the maximum amount in flight will be at most twice what is set here.
// Note that our peer may use a lower value than ours, which would reduce the maximum amount in flight.
// The smallest value of max-htlc-value-in-flight-msat and max-htlc-value-in-flight-percent will be applied when opening channels.
// If for example you open a 60 mBTC channel, eclair will set max-htlc-value-in-flight to 27 mBTC.

    max-htlc-value-in-flight-msat = 5000000000 // 50 mBTC
    max-htlc-value-in-flight-percent = 45      // 45% of the channel capacity
    max-accepted-htlcs = 30
